_docs/services/relational-database.md

```diff
@@ -117,38 +117,44 @@ In short, `cf bind-service` will provide a `DATABASE_URL` environment variable f
 
 The contents of the `DATABASE_URL` environment variable contain the credentials to access your database. Treat the contents of this and all other environment variables as sensitive.
 
-## Access the data in the database
+## Backups
+
+For shared plans (`shared-psql` and `shared-mysql`), RDS does not back up your data.
 
-There are currently two ways to access the database directly.
+For dedicated plans, RDS automatically retains daily backups for 14 days. These backups are AWS RDS storage volume snapshots, backing up the entire DB instance and not just individual databases. If you need to have a database restored using one of these backups, you can [email support](mailto:support@cloud.gov) with the following information:
```

Comment:
The runbook changes include this: `...coordinate with them on the date and time to perform the restore`. Should we specify in the customer docs a specific time period in advance to reach out for doing the restore? Maybe something like, `minimum of 48 hours in advance` reach out to support..?

Reply:
That's a good point!  Perhaps in normal circumstances we can suggest that, but if what if the customer has an immediate issue (a migration gone wrong, a compromised system) that requires attention right away?  There's no other option, correct?

Reply:
That's fair. Just wondering if - outside of an emergency - we should encourage as much advance notice as possible.

Reply:
I think that's still good to note, I'll add that language to the guides!

Reply:
I've update the language here to encourage advance notice when possible; this may be starting to touch on broader policies for the overall role and coverage of support, too, that we may need to have @RonWilliams and @eddietejeda take a look at.